src/django/api/views.py

```diff
@@ -1641,7 +1646,8 @@ def get_claimed_details(self, request, pk=None):
             )
 
             updated_claim = FacilityClaim.objects.get(pk=pk)
-
+            send_claim_update_notice_to_list_contributors(request,
```

Comment:
We should consider wrapping this notification in a try block and posting the exception to Rollbar without raising. It may be confusing for the claimant to get a failure message after we successfully committed their change to the database.

Reply:
Sounds good. This is in 8406351824b27c90638c9b7f28658ad0b3b3c007

I don't know the full range of exceptions that can get thrown when sending an email, so I just used `except Exception` for the try/except blog. In order to be parsimonious about what data get sent to Rollbar I only logged the facility claim id, although I don't know what is included in `sys.exc_info()` so it may be that that needs to go too.